modules/bim/config/locales/crowdin/js-es.yml

```diff
@@ -18,4 +18,3 @@ es:
         manage: 'Administrar modelos'
       views:
         viewer: 'Visor'
-        split: 'Visor y tarjetas'
```

Comment:
You should not edit yml translation files that are not the English ones.

Reply:
This change is deleting my previous edition (adding "split: 'Visor y tarjetas' ").